src/test/java/com/bringframework/resolver/DefaultDependencyResolverTest.java

```diff
@@ -0,0 +1,97 @@
+package com.bringframework.resolver;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+import com.bringframework.exceptions.NoSuchBeanException;
+import com.bringframework.exceptions.NoUniqueBeanException;
+import com.bringframework.registry.BeanDefinition;
+import java.util.AbstractList;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class DefaultDependencyResolverTest {
+
+  @Test
+  void noCandidateFound_NoSuchBeanExceptionThrown() {
+    var dependencyResolver = new DefaultDependencyResolver(Collections.emptyMap());
+    var desiredCandidateType = Object.class;
+    assertThrows(NoSuchBeanException.class,
+        () -> dependencyResolver.getCandidateNameOfType(desiredCandidateType));
+  }
+
+  @Test
+  void multipleCandidateFound_NoUniqueBeanExceptionThrown() {
```

Comment:
I don't remember whether we agreed on such test methods naming.

Reply:
No, we don't, but I think we have to use **camelCase** in tests naming

Reply:
renamed in camelCase and added `@DisplayName` instead to preserve readability.